LB3210.org

```diff
@@ -788,10 +788,46 @@ Route route = path("users", () ->
 #+END_SRC
 
 ** Relational databases
-*** TODO Migration management: Flyway
-TODO describe Flyway, link to code
-*** TODO Interacting with data: JOOQ
-TODO describe JOOQ, link to code
+*** Migration management
+- Relational databases change, just like code
+
+- Change management processes in the wild
+  * Schema changes just aren't allowed
+  * Schema changes require a paper form, a DBA to sign off, and a 6-month process
+  * Schema changes are done manually, live, on the production database, by a junior dev
+
+- Manage your migrations: =Flyway=
+  * Write incremental changes as scripts
+  * Trace script execution in the database itself
```

Comment:
You could consider adding that Flyway only support forward changes as opposed to other tools, e.g. LuiquidDB. The idea is to back up the database prior to flyway script execution and restore it if rollback is needed.

Reply:
Hmm let's have a talk about that. I explicitly recommend forward-only changes as the "recommended" way